shears/mine_for_gold.py

```diff
@@ -36,23 +42,57 @@ def main():
     args = parser.parse_args()
 
     print('Authenticating Reddit API connection...')
+    
+    """Authorized Reddit Instances"""
```

Comment:
Triple quotes are typically only used for docstrings.

These should probably just be regular comments, e.g.

```python
# Authenticated Reddit Instances
```

And actually, this particular comment doesn't add much value IMO, I think it's less noisy if we omit it.